dotnet-471-dev-pack/test/test.pester.ps1

```diff
@@ -0,0 +1,32 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$PackageIdentifier = ""
+)
+
+Describe "The NETFXSDK installation of pkg_ident [$PackageIdentifier] when" {
+    Context "is the SDK version 4.7" {
+        $sdk_version="4.7"

```

Comment:
One suggestion I wouldn't block merging on - we should get the version from the PackageIdentifier. Then the tests can be straight copied over to other dotnet versions and (though I don't think it applies to this plan) do a version bump of the package without needing to also update the tests code.